Misc/NEWS.d/next/Library/2017-11-19-18-42-31.bpo-10049.U91c0A.rst

```diff
@@ -0,0 +1 @@
+Added nullcontext no-op context manager to contextlib
```

Comment:
Can you please make this a complete sentence?

Reply:
I think the main missing piece here is the explanation of the user benefit. Something like:

> Added `nullcontext` no-op context manager to contextlib. This provides a simpler and faster alternative to ExitStack() when handling optional context managers.